lib/tasks/data/cities.yml

```diff
@@ -59,7 +59,7 @@
 - Mansfield
 - Middlesbrough
 - Milton Keynes
-- Newcastle-upon-Tyne
+- Newcastle upon Tyne
 - Newcastle-under-Lyme
```

Comment:
Why don't you do it for `Newcastle-under-Lyme` too?

Reply:
Same as below